File: C/BEGINNER/2165.c
#include <stdio.h>
#include <string.h>


int contarCaracteres(char *text);


int main(){
    char text[501];
    fgets(text, sizeof(text), stdin);
    int contador = contarCaracteres(text);  
    if (contador <= 140){
        printf("TWEET\n");
    } 
    else{
        printf("MUTE\n");
    }
    return 0;
}


int contarCaracteres(char *text){
    int i=0;
    while (text[i] != '\0'){
        i++;
    }
    return i-1; // REMOVENDO O '\n'
}

File: Java/BEGINNER/2165.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner keyboard = new Scanner(System.in);
        String text = keyboard.nextLine();
        if (text.length() <= 140){
            System.out.print("TWEET");
        }
        else{
            System.out.print("MUTE");
        }
        System.out.println();
    }
}
